CodeEdit/DefaultThemes/codeedit-midnight.json

```diff
@@ -0,0 +1,126 @@
+{
+  "author": "CodeEdit",
+  "distributionURL": "https://github.com/CodeEditApp/CodeEdit",
+  "version": "0.0.1",
+  "license": "MIT",
+  "type": "dark",
+  "name": "codeedit-midnight",
+  "displayName": "Midnight",
+  "description": "Midnight theme.",
+  "editor": {
+    "strings": {
+      "color": "##FF4647"
```

Comment:
Two “#” characters in this hex code, remove one.

Reply:
@austincondiff For capitalization, are you referring to the hex codes under the `terminal: {}` section?

Reply:
I am saying instead of `#af52de` change it to `#AF52DE`.

Reply:
I'm going to make these changes  to the existing themes as well then, as they also lack capitalization